assets/js/src/search.js

```diff
@@ -104,16 +106,18 @@ if (typeof require === 'function') {
             var d = me.data[r.ref];
             var href = '/resources/' + slugify(d.title);
 
-            var $result = $(
-                '<div class="search-result">' +
+            var result = '<div class="search-result">' +
                     '<a href="' + href + '">' +
                     d.title +
                     '</a>' +
                     '<p>' + truncate(d.body) + '</p>' +
-                    '</div>'
-            );
-            $el.append($result);
+                    '</div>';
+
+            r.renderedString = result;
         });
+
+        // render the first page of results
+        refreshEvents(this.results, 1);
```

Comment:
@nikolas Is this bad way of handling the rendering of the page with each search? After seeing the unit tests fail, I'm thinking that maybe this shouldn't be coupled to the DOM in this way.

Reply:
I think the way you're doing this is working well - i don't have any suggestions at the moment